### ISSUE TITLE
Getting the illegal hardware instruction (core dumped) error when using the swc cli.

### ISSUE THREAD

**Issue by yassinebridi** [swc-project/swc#1583]
**Describe the bug**
When running swc on a any typescript file, i get this error:
```
 25886 illegal hardware instruction (core dumped)
```

**Input code**
The testing project is simple.
package.json:
```json
{
  "name": "test",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "npx swc main.ts"
  },
  "devDependencies": {
    "@swc/cli": "^0.1.36",
    "@swc/core": "^1.2.52"
  }
}
```
The main.ts file:
```ts
const testTerm = "testing";
console.log("testTerm: ", testTerm);
```

**Expected behavior**
For swc to transpile and emit as expected

**Version**
The version of @swc/core: ^1.2.52

**Additional context**
I suspect this has to do with my CPU arch, so here is my `cat /proc/cpuinfo`:
```txt
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 58
model name      : Intel(R) Core(TM) i3-3110M CPU @ 2.40GHz
stepping        : 9
microcode       : 0x21
cpu MHz         : 1909.582
cache size      : 3072 KB
physical id     : 0
siblings        : 4
core id         : 0
cpu cores       : 2
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 13
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer xsave avx f16c lahf_lm cpuid_fault epb pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid fsgsbase smep erms xsaveopt dtherm arat pln pts md_clear flush_l1d
vmx flags       : vnmi preemption_timer invvpid ept_x_only flexpriority tsc_offset vtpr mtf vapic ept vpid unrestricted_guest
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit
bogomips        : 4790.34
clflush size    : 64
cache_alignment : 64
address sizes   : 36 bits physical, 48 bits virtual
power management:

processor       : 1
```

**Comment by kdy1** [comment]
Seems like avx is problematic.
I'll disable it.

**Comment by yassinebridi** [comment]
@kdy1 I supose this issues was intended to be fixed in 1.2.54,
I just tried the latest releases:
```json
"@swc/cli": "^0.1.39",
"@swc/core": "1.2.54",
```
But i think the problem still presist. I don't know how to reproduce this, is there a way to help you debug this issue?
